packages/cfd/src/Components/__tests__/cfd-financial-stp-pending-dialog.spec.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
@matin-binary @amina-deriv I have changed the naming of files as Amina to have the same naming convention with other packages. Would you please review the PR one more time ?

Reply:
Yes, that's great. better to standardise the naming convention for all the test files